### ISSUE TITLE
How enable eslint plugin to parch and lint xml files

### ISSUE THREAD

**Issue by onemen** [MiguelCastillo/Brackets-InteractiveLinter#173]
I'v recently found [eslint-plugin-mozilla](http://hg.mozilla.org/mozilla-central/file/3381f46ec15b/testing/eslint-plugin-mozilla) that is capable to lint xml files

i'v installed the plugin globally (my eslint is globally installed), and i i can use it manually to lint xml files from the command line:
`eslint --plugin [eslint-plugin-mozilla] --ext .xml .`

Is it possible to configure Brackets-InteractiveLinter to use eslint plugins and to show results for xml files?

**Comment by MiguelCastillo** [comment]
@onemen interactive linter loads and runs eslint in process; it does not call the externally available version of eslint. This has the drawback that I have to implement any particular features where eslint would otherwise handle itself as it is the case of allowing eslint to load plugins.  I am lacking some time to investigate how I can use versions of eslint in system.  I am open to suggestions and PRs. :)

**Comment by onemen** [comment]
take a look at https://github.com/fdecampredon/brackets-eslint

**Comment by MiguelCastillo** [comment]
yeah - that's one of the other way of doing it. That's the approach I describe in this other thread you got going on. https://github.com/MiguelCastillo/Brackets-InteractiveLinter/issues/166
